CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/grid/ArrangeInGridAction.java

```diff
@@ -55,7 +55,12 @@ public ArrangeInGridAction(final GraphNode context) {
 
     @Override
     public void actionPerformed(final ActionEvent e) {
-        PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+        PluginExecutor.startWith(ArrangementPluginRegistry.GRID_GENERAL)
```

Comment:
Since this is one of our most used arrangements, rather than change it i'd suggest making an `ArrangeInGridGeneralAction` to run alongside it.

Reply:
Sorry - I dont understand, are you suggesting having 2 grid arrangements in the menu ?
If so, I would think that would cause confusion to the end user.... I thought both arrangmeents were effectively the same to the end user but there may be some subleties I'm missing ?

Reply:
Sorry, it was late. `GRID_GENERAL` is a generalised version of `GRID_COMPOSITE` (which should be `GRID_COMPONENTS`?), so as long as you're defaulting to all the same default parameter values they should be equivalent. So long as that's the case we can merge this in, and probably delete `GRID_COMPOSITE`.

Reply:
Ahh yep... that was the basic theory I originally had .. but found I couldn't just delete COMPOSITE/COMPONENTS as its used in a heap of other places.

Reply:
OK - as discussed in the meeting today, we wont remove GRID_COMPOSITE under this PR, as its a standalone issue .. I'll raise a ticket for that, and as such I beleive this PR can be completed @arcturus2